test/selenium/README.md

```diff
@@ -22,18 +22,106 @@ Run the [run-tests script](https://github.com/cockpit-project/cockpit/blob/maste
+  - Then a little bit modify command what you see on the output of the ``run-tests`` command and it depends how you run the selenium grid.
+    ```
+    HUB=$HUB_IP GUEST=$GUEST_IP PORT=9991 SSH_PORT=2021 SSH_GUEST=127.0.0.2 BROWSER=chrome python3 -m avocado --show=test run  test/selenium/selenium-navigate.py
+    ```
+   - **Remember**
+     - ``GUEST`` has to be accessible from ``HUB`` via ``PORT`` number
+     - Your avocado runner has to be able to connect ``SSH_GUEST`` via ``SSH_PORT`` (what is same machine as ``GUEST`` but may have another address, because of port forwarding and ``HUB`` location)
+
+##### Description of example output
+
+  - ``PYTHONPATH=/tmp/avocado_library`` - path to libraries
+  - ``HUB=$HUB_IP_ADDR`` - ip address of your selenium grid
+  - ``GUEST=127.0.0.2`` - ip adress of cockpit machine (``HUB`` has to see ``GUEST`` cockpit port)
+  - ``PORT=9091`` - port of cockpit machine on ``GUEST`` ip adress
+  - ``SSH_PORT=22`` - ssh port number on ``SSH_GUEST``
+  - ``SSH_GUEST=10.111.113.1`` - guest ip how to connect to ssh (``GUEST`` has to be able to connect ``SSH_GUEST:SSH_PORT``)
+  - ``BROWSER=chrome`` - name of browser to use
+  - ``timeout 297`` - timeout to kill the ``avocado`` process (could be remove in case of manual rescheduling)
+  - ``python3 -m avocado run --show-job-log`` - avocado test scheduler (in case you have newer avocado (>70.0) there is little bit different sytanx ``python3 -m avocado --show=test run ``)
+  - ``/tmp/avocado_tests/selenium-navigate.py:NavigateTestSuite.testNavigateNoReload`` name of test, it consists of ``filename_path:Class_name.tests_method_name``
+  - ``2>&1`` - could be remove in case of manual rescheduling
+
 ## Hacking
 
 ### Selenium tests
-[run-tests script](https://github.com/cockpit-project/cockpit/blob/master/test/selenium/run-tests) can change it's behavior by the environment variables specified bellow.
+[run-tests script](https://github.com/cockpit-project/cockpit/blob/master/test/selenium/run-tests) can change its behavior by the environment variables specified bellow.
 
 #### Where cockpit is running
-Defines where is the cockpit instance that you want to test. This cannot be used together with ``TEST_OS`` variable.
+Defines where is the cockpit instance that you want to test. This cannot be used together with the ``TEST_OS`` variable.
 
  - ``GUEST`` (default: ``localhost``) - defines hostname or IP of the machine where cockpit-ws is running
    - This machine has to have enabled ssh for execution remote commands for purpose of ``self.machine.execute``
  - ``PORT`` (default: ``9090``) - defines the port where cockpit-ws component accepts connections in GUEST machine
  - ``URL_BASE`` (default: ``http``) - defines what protocol to use, http or https.
+ - ``SSH_GUEST`` (default: same as ``GUEST``) - define name or ip of cockpit machine, it is important for ``run-tests`` scheduler for debugging, where is used port forwarding
```

Comment:
s/of cockpit/of the cockpit/

s/for\ \`\`run-tests`` scheduler/for the\ \`\`run-tests`` scheduler/

Reply:
Actually if you can go over the whole change and make sure you use articles it would be great :) I think I might have missed some.